src/components/features/facility/chiropractor/chiropractor.php

```diff
@@ -1,6 +1,19 @@
-<div class="flex justify-start items-center gap-10">
+<!-- desktop -->
+<div class="flex phone:hidden justify-start items-center gap-10">
   <img src="<?php echo $absoluteUriPath ?>/asset/chiropractor/chiropractor.png" />
```

Comment:
[imo]
`desktop`と`phone`のhtmlの差分が少ないからclassでのやりくりでOK！

```suggestion
<div class="flex phone:flex-col justify-start items-center gap-10">
  <img class="phone:w-[calc(100%-32px)]" src="<?php echo $absoluteUriPath ?>/asset/chiropractor/chiropractor.png" />
```